docs/baseline-protocol-code/branch-naming.md

```diff
@@ -0,0 +1,23 @@
+# Branch Naming Standard
+
+## Main
+
+*main* - main development branch, feature and release branches branched from it, changes only through the PR process.
+
+## Feature
+
+*feature/XYZ-this-is-a-new-feature-branch* - in case related issue XYZ exists
+
+*feature/BLIP-#-repo-branch-standards* - in case related BLIP-# exists
+
+*feature/this-is-a-new-feature-branch* - in case no related item exists
```

Comment:
we should probably discourage opening branch without issue or blip? is there a valid case for feature like this?

Reply:
Removed that option. Lets always go with existing issue or blip.